docs/INSTALLATION.md

```diff
@@ -34,3 +34,7 @@ by executing the ``lfe`` binary:
 $ ./bin/lfe
 ```
 
+The dependencies for Ubuntu 14.04 were:
```

Comment:
I think this would be better as something like:

> For example, on Ubuntu 14.04:
>
> ```bash
> $ sudo apt-get install build-essentials git erlang erlang-base-hipe
> ```

or whatever the real dependencies are after the comment about "build essentials", `git` and `erl`.